### ISSUE TITLE
is this even still a wip

### ISSUE THREAD

**Issue by Jeydin31** [Another-19/genshin-impact-wishing-simulator#1]
pretty nice wip for something that was last edited 24 days ago

**Comment by Another-19** [comment]
yes, I am reworking the whole project into a discord bot instead, is there a problem with anything?

**Comment by Jeydin31** [comment]
oh, i didnt know that 0-0 thats pretty poggers
